lua/nvim-tree/commands.lua

```diff
@@ -12,7 +9,7 @@ local CMDS = {
       complete = "dir",
     },
     command = function(c)
-      api.tree.open({ path = c.args })
+      require("nvim-tree.api").tree.open({ path = c.args })
```

Comment:
Initial thought was to use a common function like
```lua
local function api()
  require("nvim-tree.api")
end
```
however that would just obscure things.
The inline require make it really clear what's going on.